### DISCUSSION TITLE
Support for `# pyright: standard`

### DISCUSSION THREAD

**Discussion by kalekseev** [microsoft/pyright#9085] [Ideas]
Hi, it's not possible to change type check strictness to standard using `#pyright: standard`, pyright show error:

> "standard" is an unknown directive for pyright comment; expected "strict" or "basic"

I wonder is there any reason why it's not implemented? I'm trying to migrate huge codebase to strict mode right now and I often find myself in a situation where whole module can be checked in strict mode except some low impact files or tests. So what I would like to do is to add this module path to strict = [ ] setting but keep some files on standard level of type check thus need `#pyright: standard`

**Comment by erictraut** [comment]
Ah, this is a bug. This directive should work. Pyright originally had only "strict" and "basic" modes. When I added "standard", I forgot to enable it for `#pyright` comments. 

This will be fixed in next week's release. Thanks for reporting the issues.
